Taskfile.yml

```diff
@@ -277,7 +277,7 @@ tasks:
         --mode=diff-normal
         --handler=pg
         --postgresql-url=postgres://username@127.0.0.1:5432/ferretdb
-        --test-records-dir=tmp/records
+        --test-records-dir=./tmp/records
```

Comment:
Please revert those path changes. It will work as it is.

Reply:
This is temporary. My settings create files in an inconvenient place for debugging